ktor-server/ktor-server-plugins/ktor-server-cors/jvmAndNix/src/io/ktor/server/plugins/cors/CORSConfig.kt

```diff
@@ -96,6 +96,11 @@ public class CORSConfig {
      */
     public var allowCredentials: Boolean = false
 
+    /**
+     * If present allows any origin matching any of the predicates.
+     */
+    public val originPredicates: MutableList<(String) -> Boolean> = mutableListOf()
```

Comment:
can be private

Reply:
I really mirrored what I saw for headers, do you see any reason why `headerPredicates` is public?

Reply:
No, there is no reason, and it should be internal. I will clean up it in the next major release. We now try to avoid exposing mutable collections in public API.

Reply:
Would you rather have `originPredicates` still be `public` here for coherence with the current major or make it `private` immediately to prevent any futur user migration?

Reply:
I'll prefer it to be private (internal, actually)

Reply:
Fixed. I also ran `./gradlew apiDump` again before committing.